blog-article.html

```diff
@@ -43,7 +43,7 @@ <h1 class="logo">
             </nav>
             <div class="tools">
                 <form class="search">
```

Comment:
Please add method to all forms
<img width="1030" alt="image" src="https://github.com/user-attachments/assets/e6bf2d4a-6b26-459d-a7ce-e2f5fefdf62a">

Reply:
Done